lib/features/create_transaction/screens/choose_amount_slider_screen.dart

```diff
@@ -0,0 +1,253 @@
+// ignore_for_file: prefer_const_constructors, prefer_const_literals_to_create_immutables
+import 'dart:developer';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app_kids/features/create_transaction/cubit/create_transaction_cubit.dart';
+import 'package:givt_app_kids/features/create_transaction/models/transaction.dart';
+import 'package:givt_app_kids/features/create_transaction/screens/success_screen.dart';
+import 'package:givt_app_kids/features/profiles/cubit/profiles_cubit.dart';
+import 'package:givt_app_kids/shared/widgets/wallet.dart';
+
+import 'package:givt_app_kids/widgets/back_button.dart' as custom_widgets;
+
+class ChooseAmountSliderScreen extends StatefulWidget {
+  static const String routeName = "/choose-ammount-slider-bloc";
+
+  const ChooseAmountSliderScreen({Key? key}) : super(key: key);
+
+  @override
+  State<ChooseAmountSliderScreen> createState() =>
+      _ChooseAmountSliderScreenState();
+}
+
+class _ChooseAmountSliderScreenState extends State<ChooseAmountSliderScreen> {
+  late final ProfilesCubit _profilesCubit;
+
+  @override
+  void initState() {
+    _profilesCubit = context.read<ProfilesCubit>();
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    // final organisation =
+    //     ModalRoute.of(context)?.settings.arguments as Organisation;
+
+    return BlocProvider<CreateTransactionCubit>(
+      create: (BuildContext context) =>
+          CreateTransactionCubit(profilesCubit: _profilesCubit),
+      child: SafeArea(
+        child: BlocConsumer<CreateTransactionCubit, CreateTransactionState>(
+          listener: (context, state) {
+            log('auth state changed on $state');
+
+            if (state is CreateTransactionErrorState) {
+              log(state.errorMessage);
+              ScaffoldMessenger.of(context).showSnackBar(
+                SnackBar(
+                  content: Text(
+                    "Cannot create transaction. Please try again later.",
+                    textAlign: TextAlign.center,
+                  ),
+                  backgroundColor: Theme.of(context).errorColor,
+                ),
+              );
+            } else if (state is CreateTransactionSuccessState) {
+              Navigator.of(context).pushNamed(SuccessScreen.routeName);
```

Comment:
I am thinking we can re-fetch profiles also here, so the app gets new data while the user is on the success screen, so there is no 'wait'.

Reply:
good idea, thanks, shall we add it when switch on new flow?